tests/Unit/Evolution/Systems/ScalarAdvection/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY Test_ScalarAdvection)
+
+set(LIBRARY_SOURCES
+  # BoundaryConditions/...
+  # BoundaryCorrections/...
+  Test_Tags.cpp
+  )
+
+add_test_library(
+  ${LIBRARY}
+  "Evolution/Systems/ScalarAdvection"
+  "${LIBRARY_SOURCES}"
+  ""
+  )
+
+target_link_libraries(
+  ${LIBRARY}
+  PUBLIC
+  ScalarAdvection
+  # ScalarAdvectionAnalyticData
```

Comment:
Can you remove the CMake comments or does CMake give you errors if you do? I think it would be better to add the libraries uncommented as you need them. Same with the `# BoundaryConditions/...` and `# BoundaryCorrections/...` above

Reply:
Comments deleted :)

Reply:
Yep, comments deleted :)